pkg/core/util.go

```diff
@@ -94,6 +94,18 @@ func deployNativeContracts(magic netmode.Magic) *transaction.Transaction {
 	return tx
 }
 
+func commiteeFromConfig(cfg config.ProtocolConfiguration) ([]*keys.PublicKey, error) {
+	validators := make([]*keys.PublicKey, len(cfg.StandbyCommitee))
+	for i := range validators {
+		pubKey, err := keys.NewPublicKeyFromString(cfg.StandbyCommitee[i])
+		if err != nil {
+			return nil, err
+		}
+		validators[i] = pubKey
+	}
+	return validators, nil
+}
+
 func validatorsFromConfig(cfg config.ProtocolConfiguration) ([]*keys.PublicKey, error) {
```

Comment:
I think it can reuse `commiteeFromConfig()` now.

Reply:
Or we can even share the actual slice between committee and validators.

Reply:
Fixed.